o/openapi-schema-validator/openapi-schema-validator_ubi_8.3.sh

```diff
@@ -0,0 +1,67 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : openapi-schema-validator
+# Version               : 0.1.4
```

Comment:
please specify all the versions with which this script has been tested

Reply:
Done and merged ,Pls check now